www/obs/obs.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
```

Comment:
Could you please make this a SCSS file, add it to the lint script, and add it to the build script?